pkg/apis/operator/v1alpha1/tektonhub_defaults.go

```diff
@@ -24,11 +24,11 @@ import (
 func (th *TektonHub) SetDefaults(ctx context.Context) {
 
 	if th.Spec.Db.DbSecretName == "" {
-		th.Spec.Db.DbSecretName = "db"
+		th.Spec.Db.DbSecretName = "tekton-hub-db"
```

Comment:
let us move the string `tekton-hub-` into a project level constant or a package level constant.